Doc/library/lzma.rst

```diff
@@ -349,6 +349,8 @@ options. Valid filter IDs are as follows:
   * :const:`FILTER_ARMTHUMB`
   * :const:`FILTER_POWERPC`
   * :const:`FILTER_SPARC`
+  * :const:`FILTER_ARM64`
```

Comment:
All these constant names need ! added, as in  `!FILTER_X86`, to suppress the search for a link and the warnings.

Reply:
OK. Added.